lib/src/clojure_lsp/kondo.clj

```diff
@@ -131,27 +131,28 @@
       (assoc-in [:config :linters :unresolved-var :report-duplicates] true))))
 
 (defn ^:private custom-lint-project!
-  [db* {:keys [analysis config] :as kondo-ctx}]
```

Comment:
Looking through clojure-lsp.kondo, there are more places that don't really need `db*`, and would be happy with `db`. It's tricky because the custom-lint calls are all callbacks that should deref the db when they're called, not when they callback is constructed. Anyway, I'll plan to take another look at this file some time.